builders/deploy.yaml

```diff
@@ -85,9 +52,9 @@ steps:
   args:
     - functions
     - deploy
-    - ProvideVerificationCode
+    - RegisterNotificationAfterMath
     - --source=.
-    - --trigger-http
+    - --trigger-topic=notification-registered
```

Comment:
this topic is created by hand? I'll add it to terraform in another PR:)

Reply:
It's created automa[tg]ically when it doesn't exist.

Reply:
aha!